.github/CHANGELOG.md

```diff
@@ -182,6 +182,27 @@
 * Probability methods are handled by `QubitDevice` and device method
   requirements are modified to simplify plugin development.
   [(#573)](https://github.com/XanaduAI/pennylane/pull/573)
+  
+* The ``QAOAEmbedding`` now uses the new ``MultiRZ`` gate as a ``ZZ`` entangler, 
+  which changes the parameter convention: While 
+  previously, the ``ZZ`` gate in the embedding was implemented as
+  
+  .. code-block:: python
+  
+    CNOT(wires=[wires[0], wires[1]])
+    RZ(2 * parameter, wires=wires[0])
+    CNOT(wires=[wires[0], wires[1]]) 
+  
+  the ``MultiRZ`` corresponds to 
+
+  .. code-block:: python
+  
+    CNOT(wires=[wires[0], wires[1]])
```

Comment:
I believe that there's going to be an extra subtlety here: the target and the control wires need to be swapped.

This is the output of the decomposition of `MultiRZ` (which seems to be correct to me):
```python
In [10]: a = qml.ops.MultiRZ(weights, wires=[0,1])
In [11]: a.decomposition(0, wires=[0,1])                                        
Out[11]: [CNOT(wires=[1, 0]), RZ(0, wires=[0]), CNOT(wires=[1, 0])]
```
That is `1` is the control whereas `0` is the target qubit here.

This is also something connected to the code that is being changed here:
```python
def zz(weight, wires):	
    CNOT(wires=wires)	
    RZ(2 * weight, wires=wires[0])	
    CNOT(wires=wires)
```
would instead need to have been
```python
def zz(weight, wires):	
    CNOT(wires=wires)	
    RZ(2 * weight, wires=wires[1])	#<--- RZ acting on the wire that was the target of the CNOT
    CNOT(wires=wires)
```
With the original version, the `CNOT` before and after the `RZ` seems to make no difference. However, let me know if you think I've missed something here.

Reply:
Great, thanks for being so thorough! 

That is indeed not good, the bug in the previous code. Shall I open an issue, or do we just fix it and leave it at that?

Reply:
I updated the changelog to mention the bug.

Reply:
No worries at all! Glad if we are both convinced :smile: I think it's fine to fix it with this PR :slightly_smiling_face:  perhaps also thinking through if this might have an effect on any other parts of the codebase (from what I saw the QAOA tutorial had its own implementation)

Reply:
Yes, I will check the tutorial asap!

Reply:
I also know now why the test didn't pick it up, I made a logical error in testing the ZZ gate previously, was actually testing the feature input instead...